src/plugins/console/public/application/hooks/use_send_current_request_to_es/use_send_current_request_to_es.ts

```diff
@@ -31,17 +31,40 @@ export const useSendCurrentRequestToES = () => {
     services: { history, settings, notifications, trackUiMetric },
   } = useServicesContext();
 
+  const { requestInFlight } = useRequestReadContext();
   const dispatch = useRequestActionContext();
 
   return useCallback(async () => {
+    if (requestInFlight) {
+      notifications.toasts.addWarning({
+        title: i18n.translate('console.requestAlreadyInFlightTitle', {
+          defaultMessage: 'Cannot send request right now.',
+        }),
+        toastLifeTimeMs: 3000,
+      });
+      return;
+    }
     dispatch({ type: 'sendRequest', payload: undefined });
     try {
       const editor = registry.getInputEditor();
       const requests = await editor.getRequestsInRange();
       if (!requests.length) {
         dispatch({
           type: 'requestFail',
-          payload: { value: 'No requests in range', contentType: 'text/plain' },
+          payload: {
```

Comment:
I feel like the way this is represented in the UI isn't particularly useful. I didn't actually send a request to an endpoint, and there was no real response nor elapsed time, so this information in the top-right corner isn't very useful:

![image](https://user-images.githubusercontent.com/1238659/73793339-09c93a00-475b-11ea-856a-8d1607b76a3d.png)

Can we make this more useful? How about we preserve the existing information from the prior request (i.e. the response payload in the output pane as well as the status and elapsed time badges), and instead just show a neutral toast to let the user know that they need to move their cursor inside of a request in order to execute it?